packages/cli/src/MCP.ts

```diff
@@ -0,0 +1,183 @@
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js'
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js'
+import type { PrismaConfigInternal } from '@prisma/config'
+import { Command, link } from '@prisma/internals'
+import { spawn } from 'child_process'
+import { z } from 'zod'
+
+import { createHelp } from './platform/_lib/help'
+
+function spawnAsPromise(
```

Comment:
You can use `execa` (we already depend on it, albeit an old version) so you don't have to re-implement this manually